geode-redis/src/integrationTest/java/org/apache/geode/redis/general/ExpireIntegrationTest.java

```diff
@@ -37,7 +37,8 @@
 
 public class ExpireIntegrationTest {
 
-  private static Jedis jedis;
+  public static Jedis jedis;
+  public static int REDIS_CLIENT_TIMEOUT = 10000000;
   private static GeodeRedisServer server;
   private static GemFireCache cache;
   private static Random rand;
```

Comment:
`rand` is unused in the file.  And `port` below could be local to the `setUp` method

Reply:
Good catch, "rand" was leftover from earlier tests.